### DISCUSSION TITLE
Accelerometer with display demo

### DISCUSSION THREAD

**Discussion by ericye16** [google/comprehensive-rust#576] [Show and tell]
While working on https://google.github.io/comprehensive-rust/exercises/bare-metal/compass.html, I decided I wanted to visualize the accelerometer readings, so I created this short demo using the display: https://photos.app.goo.gl/q2jAhkC2kAQebSG58

Code here: https://gist.github.com/ericye16/38290804e7c4a26060f05aee2fd7018b

**Comment by mgeisler** [comment]
Awesome, thanks for posting this! :smile:
